src/cli/bin.js

```diff
@@ -15,6 +15,8 @@ updateNotifier({
 yargs
   .commandDir('commands')
   .demand(1)
+  // Temporary alias of {add,cat,get}
+  .commandDir('commands/files')
```

Comment:
It is more the other way around. `{add, cat, get}` are temporary alias of `{ fileas add, files cat, files get}`